app/assets/stylesheets/modules/_family.scss

```diff
@@ -0,0 +1,149 @@
+.family {
+  // @include span-columns(12 of 12);
+  width: 95%;
```

Comment:
Properties should be ordered background-color, border-radius, margin-left, padding, width